api/spec/requests/spree/api/users_controller_spec.rb

```diff
@@ -144,6 +144,21 @@ module Spree
         expect(response.status).to eq(422)
         expect(json_response).to eq({ "error" => "Cannot delete record." })
       end
+
+      it "returns distinct search results" do
+        uuser = create(:user, email: 'distincttest@solidus.com')
```

Comment:
Small typo here, should be `user`?

Reply:
It was meant to be an abbreviation of unique_user. i have changed the spec it to be more precise.

Reply:
Ah, got it! Yeah, I think the full version is more easily understandable.